ods_ci/tasks/Resources/RHODS_OLM/uninstall/oc_uninstall.robot

```diff
@@ -39,5 +39,7 @@ Wait Until Project Is Deleted
         ${project_exists}=  Run Keyword and return status
         ...  Oc Get  kind=Namespace  field_selector=metadata.name=${project}
         Exit For Loop If     not ${project_exists}
-
+   END
+   IF    ${project_exists}
+        Fail    Project ${project} has not been deleted!
```

Comment:
could you update the message. Something like Project has not been deleted after ${timeout} attempts

Reply:
Done, thanks!